frontend/src/pages/Home.tsx

```diff
@@ -1,5 +1,15 @@
+import { useContext } from "react"
+import { LoginContext } from "../Components/LoginManager/LoginManager"
+
 const Home = () => {
-  return <h1>Home</h1>
+  const { roleType } = useContext(LoginContext)
+  console.log(roleType)
+  return (
+    <div>
+      <h1>Home</h1>
+      <h1>Role Type: {roleType}</h1>
```

Comment:
do we want to keep this in the PR? I had assumed it was for manual testing?
If we do keep it I think it should only be visible to admins tbh

Reply:
yes it was just to check that the role type is passed everywhere and needs to be removed (line 10)

Reply:
I'm happy to approve this then under the condition you remove that line before merging